src/js/controller/model.js

```diff
@@ -407,7 +406,8 @@ define([
         };
 
         this.autoStartOnMobile = function() {
-            return this.get('autostart') && utils.isMobile() && !this.get('mobileSdk');
+            return this.get('autostart') && utils.isMobile() && !this.get('mobileSdk') &&
+                (!this.get('advertising') || this.get('advertising').autoplayadsmuted);
```

Comment:
Did you sync up with @esboyd on the config name? Some thoughts: 

- If this is in the advertising block do we need to include the word `ads` in it? 
- If we say `autostart` for the media, we should use the same term for ads and not use `autoplay`. 
- If looking at just the config, there's nothing in the name that implies that mute on autostart is mobile only.
- unless I'm mistaken, I think we're moving towards camelCase for the config names. 

Maybe something like `muteOnMobileAutostart` or `muteOnMobilePlay`?